src/Data/String/Conversions.hs

```diff
@@ -25,6 +27,7 @@ module Data.String.Conversions (
     SBS,
     LazyByteString,
     LBS,
+    Builder,
```

Comment:
`Builder` doesn't really make clear that it's a builder for bytestrings. Maybe we should have two type synonym called `ByteStringBuilder` and `BSBuilder` (or `BSB`)?